### PR TITLE
Home work week 3

### PR DESCRIPTION
**Pull request by v-n7k**

### CODE REVIEW

File: test/loanManager.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+    it('should not allow not owner to repay', () => {
+        const borrower = accounts[6];
+        const hacker = accounts[9];
+        const value = 900;
+        const errorStr = 'You are not the loan owner';
+        return Promise.resolve()
+            .then(() => loanManager.borrow(value, {from: borrower}))
+            .then(() => loanManager.repay.call(borrower, value, {from: borrower}))
+            .then(result => assert.deepEqual([false, errorStr], result))
```
Comment (Purka) [inline test/loanManager.js:54]: Why do you use assert.deepEqual() instead assert.isFalse() ?

File: contracts/LoanManager.sol [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    function borrow(uint amount) public returns (bool) {
+        loans[msg.sender] = add(loans[msg.sender], amount);
+        return true;
+    }
+
+
+    function repay(address borrower, uint amount) public returns(bool, string){
+        if (msg.sender != owner){
+            return (false, "You are not the loan owner");
+        } else if (loans[borrower] < amount){
```
Comment (Purka) [inline contracts/LoanManager.sol:18]: Do you cover it in your tests?

File: test/loanManager.js (hunk 1 above)
Comment (v-n7k) [reply test/loanManager.js:54]: I'm checking equal two arrays

File: contracts/LoanManager.sol (hunk 2 above)
Comment (volodymyr-nakvasiuk) [reply contracts/LoanManager.sol]: Test: "should not allow not owner to repay"

**Pull request closed without merging**